banzai/tests/e2e-k8s.yaml

```diff
@@ -16,7 +16,7 @@ spec:
     # When the Pod is initialized, copy all files within the container at path
     # /archive/engineering into the empty data volume mounted at /data
     - name: banzai-data
-      image: docker.lco.global/banzai-e2e-data:1.3.0
+      image: docker.lco.global/banzai-e2e-data:caca61c040ed
```

Comment:
TODO: Update this when new banzai-e2e-data is tagged

https://github.com/LCOGT/banzai-e2e-data/pull/9